server/src/test/java/io/spine/server/entity/RecordBasedRepositoryTest.java

```diff
@@ -301,6 +295,8 @@ void noEntitiesIfEmpty() {
         }
     }
 
+    protected abstract E entity(I id);
```

Comment:
Please move to the `protected` methods section (see above). Also, I don't understand why it is not an overload of `createEntity() : E` method, which already exists above. `createEntity(id) : E` would look much more consistent.